### DISCUSSION TITLE
Should the Quartz.Web project be deprecated/purged in v4?

### DISCUSSION THREAD

**Discussion by jafin** [quartznet/quartznet#1970] [Q&A] [answered]
Just looking through some history and the Quartz.Web project hasn't seen much love or maintenance in the 3.x lifetime.  (it has a framework of netcoreapp2.0)

There is also a Quartz.Examples.AspNetCore which is maintained.

Should Quartz.Web be dropped for v4?  Or is there a desire from anyone to see it active?  If so what exactly is the intent of the project compared to the Example.AspNetCore?

As it stands now, it's get hits on a lot of security vulnerabilities (mainly due to outdated framework etc) on snyk.
2 Critical, 26 High
![image](https://user-images.githubusercontent.com/127927/227716437-bf63943d-2996-4f65-b8f1-8c9a83a679c7.png)

**Comment by lahma** [comment, marked as answer]
I've cleaned up the project so it should no longer show up in alerts.  Quartz.Web is still a playground and was source for example the new HTTP API which was derived (and improved) from ideas of the playground. Maybe at some point can update the UI prototype to use the new HTTP API and see if/what improvements could be made.

**Reply by jafin** [reply]
Sounds good, thanks @lahma
